src/Bellatrix.Web/Bellatrix.Web.csproj

```diff
@@ -15,7 +15,7 @@
 	<ItemGroup>
 		<PackageReference Include="HtmlAgilityPack" Version="1.11.43" />
 		<PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
-		<PackageReference Include="Selenium.WebDriver" Version="4.4.0" />
+		<PackageReference Include="Selenium.WebDriver" Version="4.5.1" />
```

Comment:
as we spoke please open the BELLATRIX.sln and check that all projects are updated properly that everything is buildable and that a few tests in the getting started web are running